cpp/ql/test/TestUtilities/InlineExpectationsTest.qll

```diff
@@ -233,7 +233,9 @@ private string expectationPattern() {
   exists(string tag, string tags, string value |
     tag = "[A-Za-z-_][A-Za-z-_0-9]*" and
     tags = "((?:" + tag + ")(?:\\s*,\\s*" + tag + ")*)" and
-    value = "((?:\"[^\"]*\"|'[^']*'|\\S+)*)" and
+    // In Python, we allow both `"` and `'` for strings, as well as the prefixes `bru`.
+    // For example, `b"foo"`.
+    value = "((?:[bru]*\"[^\"]*\"|[bru]*'[^']*'|\\S+)*)" and
```

Comment:
Would it be sufficient to just have `[bru]?` instead? Or does Python actually need arbitrarily long sequences of `[bru]`? 😱

Reply:
Yes, you can have `br"foo\."` for a raw bytes-string, that would otherwise have to be written as `b"foo\\."`. I think single letter prefixes are used 99.9% of the time, but I've seen multi prefix a couple of times in my life.

I guess one obvious omission is f-string support like `f"self.foo={self.foo}"`. However, since f-strings have more semantic meaning than just a string constant, I'm not sure we want support for that. So trying to KISS, and saving that discussion for when/if we ever need it.